pkg/interface/groups/src/js/components/lib/contact-card.js

```diff
@@ -200,14 +235,6 @@ export class ContactCard extends Component {
         }
         break;
       }
-      case 'removeAvatar': {
-        this.setState({ awaiting: true, type: 'Removing from group' }, (() => {
-            api.contactEdit(props.path, ship, { avatar: null }).then(() => {
-              this.setState({ awaiting: false });
-            });
-        }));
-        break;
-      }
```

Comment:
Note that we had two cases for the removeAvatar case introduced in 15404e0989683151ffe0fe8daf6323e9059863e6, heh. We just remove the stubbed one I wrote originally here.